src/test/java/org/bytedeco/javacpp/AdapterTest.java

```diff
@@ -133,28 +145,46 @@
         String textStr1 = "This is a normal ASCII string.";
         String textStr2 = testStdString(textStr1);
         assertEquals(textStr1, textStr2);
+        String textStr3 = testStdString2(textStr1);
+        assertEquals(textStr1, textStr3);
 
         BytePointer textPtr1 = new BytePointer(textStr1);
         BytePointer textPtr2 = testStdString(textPtr1);
         assertEquals(textStr1, textPtr1.getString());
         assertEquals(textStr1, textPtr2.getString());
+        BytePointer textPtr3 = testStdString2(textPtr1);
+        assertEquals(textStr1, textPtr3.getString());
+
+        CharPointer textCharPtr1 = new CharPointer(textStr1);
+        assertEquals(textStr1, textCharPtr1.getString());
+
+        IntPointer textIntPtr1 = new IntPointer(textStr1);
+        assertEquals(textStr1, textIntPtr1.getString());
 
         if (Loader.getPlatform().startsWith("windows")) {
             // UTF-16
-            CharPointer textCharPtr1 = new CharPointer(textStr1);
             CharPointer textCharPtr2 = testStdWString(textCharPtr1);
-            assertEquals(textStr1, textCharPtr1.getString());
             assertEquals(textStr1, textCharPtr2.getString());
+            CharPointer textCharPtr3 = testStdWString2(textCharPtr1);
+            assertEquals(textStr1, textCharPtr3.getString());
         } else {
             // UTF-32
-            IntPointer textIntPtr1 = new IntPointer(textStr1);
             IntPointer textIntPtr2 = testStdWString(textIntPtr1);
-            assertEquals(textStr1, textIntPtr1.getString());
```

Comment:
Why are these asserts removed?

Reply:
It's moved in higher scope (see line 162) so that textIntPtr1 and textCharPtr1 could be used with testStdU32String and testStdU16String.